README.md

```diff
@@ -797,6 +797,82 @@ In either case:
     end
     ```
 
+### Nested conditionals
+
+* <a name="no-nested-conditionals"></a>
+  Avoid the use of nested conditionals for flow of control. 
+  ([More on this][avoid-else-return-early].) <sup>[[link](#no-nested-conditionals)]</sup>
+
+  Prefer a guard clause when you can assert invalid data. A guard clause
+  is a conditional statement at the top of a function that bails out as
+  soon as it can. 
+  
+  The general principles boil down to:
+  * Return as soon as you know your method cannot do any more meaningful work
+  * Reduce indentation by using `if`/`unless`/`return` instead of a top-level 
+  `if`/`else`
+  * Try keep the “meat” of your method at the lowest indentation level.
+
+  ```ruby
+  # bad
+  def compute_thing(thing)
+    if thing[:foo]
+      update_with_bar(thing[:foo])
```

Comment:
thanks for the suggestion. i will rework this with some more realistic examples.